tests/core/pyspec/eth2spec/test/bellatrix/block_processing/test_process_execution_payload.py

```diff
@@ -109,15 +109,15 @@ def run_success_test(spec, state):
     yield from run_execution_payload_processing(spec, state, execution_payload)
 
 
-@with_all_phases_from_except(BELLATRIX, [EIP7732])
+@with_all_phases_from_except(BELLATRIX, [GLOAS])
```

Comment:
Give that this exclusion of Gloas comes from the separation of beacon block and payload, I think `@with_all_phases_from_to(BELLATRIX, GLOAS)` is correct. Otherwise, it would break in H* fork. This holds for other test files, too.